### DISCUSSION TITLE
Update failed

### DISCUSSION THREAD

**Discussion by h4rvey-g** [ObsidianPublisher/obsidian-github-publisher#183] [Q&A — Template]
I have set ` EXCLUDED-FILES: 'overrides/partials/comments.html'` in my `update.yml`. But it fails to update. Here is the log:
```
Run zip -d release.zip overrides/partials/comments.html
  zip -d release.zip overrides/partials/comments.html
  unzip -o release.zip
  rm release.zip
  shell: /usr/bin/bash -e {0}

zip error: Nothing to do! (release.zip)
Error: Process completed with exit code 1[2](https://github.com/h4rvey-g/Notes-blog/actions/runs/5686925432/job/15414551554#step:5:2).
```

**Comment by Mara-Li** [comment]
1. First, don't open discussion for this kind of problem. It's better to open a issue in [GithubPublisher/actions](https://github.com/ObsidianPublisher/actions).
2. You don't need to set the comments in the excluded files sections, as the comments are already excluded.
